wp-content/themes/engage/assets/scss/components/_widget.scss

```diff
@@ -29,34 +29,59 @@
 		color: $black-grey;
 	}
 
-	.constantcontactwidget_form {
-		p {
-			margin-bottom: 1rem;
-			color: $black-grey;
-		}
+	.textwidget {
+		.wpcf7-form {
+			margin: 0;
 
-		input[type="text"] {
-			font: inherit;
-			margin-top: 1rem;
-			width: 100%;
+			.newsletter-prompt {
+				margin-bottom: 1.6rem;
+				color: $black-grey;
+			}
 
-			@include media($tablet) {
+			.newsletter-input {
+				font: inherit;
```

Comment:
Does this `font: inherit;` make a difference here?

Reply:
Yes, it changes the size